src/settings.ts

```diff
@@ -511,4 +511,4 @@ function createSnippetsEditor(content: string, extensions: Extension[]) {
     });
 
     return view;
-}
\ No newline at end of file
+}
```

Comment:
Oh whoops, this change was not intended. I guess Neovim saved the file indirectly and added a newline here. Is that okay?

Reply:
That's fine! Would you mind moving the `\t` so that it's located just after `\\n`? I think it makes sense for `:` and `;` to be sat next to each other.